api/v2_vulnerabilities_test.go

```diff
@@ -80,6 +80,18 @@ func TestV2Vulnerabilities_Hosts_SearchAllPages_EmptyData(t *testing.T) {
 	assert.Equal(t, 0, len(response.Data))
 }
 
+func TestV2VulnerabilitiesFilterSingleVulnID(t *testing.T) {
+	var (
+		response api.VulnerabilitiesContainersResponse
+		vulnID   = "CVE-2017-12670"
+	)
+
+	json.Unmarshal([]byte(mockVulnerabilitiesContainersResponse()), &response)
```

Comment:
What if there are more than one vulnerability information about the same CVE id?

Reply:
I tried out "dirty image" that has this CVE ID four times and it doesn't really display the information correctly:
```
lacework vulnerability ctr show sha256:77b2d2246518044ef95e3dbd029e51dd477788e5bf8e278e418685aabc3fe28a CVE-2022-1271
```

Reply:
I've updated this to repeat the table when the are multiple results of the same cve-id.